README.md

```diff
@@ -162,7 +162,11 @@ mqtt_pub -i "a:${IOT_ORG_ID}:client_pub" -u "${IOT_API_KEY}" -P "${IOT_AUTH_TOKE
 }'
 ```
 
-### 2. Upload Actions
+### 2. Deploy MQTT Feed
+
+Install the MQTT package/feed found in the openwhisk-package-mqtt-watson submodule [here](openwhisk-package-mqtt-watson). This "feed" enables OpenWhisk to subscribe to one or more MQTT topics and invoke actions in response to incoming messages. To see more on how feeds work with IBM Cloud Functions, please visit these [documents](https://github.com/apache/incubator-openwhisk/blob/master/docs/feeds.md)
```

Comment:
the target of [here] will need to be something like https://github.com/kkbankol-ibm/openwhisk-package-mqtt-watson/tree/c9938971eab75b0098323fe008c116667193cdb2 to work correctly